.github/workflows/therock-test-packages-multi-node.yml

```
@@ -59,18 +67,27 @@ jobs:
           FETCH_ARTIFACT_ARGS: "--rccl --tests"
           IS_PR_FROM_FORK: ${{ github.event.pull_request.head.repo.fork }}
 
-      # The following step leverages slurm to run multi node rccl tests on the slurm mi350x cluster.
-      # salloc will hold 4 nodes while the commands inside the block run. After the block completes, salloc automatically releases the nodes.
+      ## The following step leverages slurm to run multi node rccl tests on the slurm mi350x cluster.
+      ## salloc will hold 4 nodes while the commands inside the block run. After the block completes, salloc automatically releases the nodes.
       - name: Test gfx950
         if: ${{ inputs.amdgpu_families == 'gfx950-dcgpu' }}
         run: |
+          # Determine config file based on NIC count
+          if [[ "${ENABLED_NICS}" == "8nic" ]]; then
+            CONFIG_FILE="./input/mi350_8nic_config.json"
+          else
+            CONFIG_FILE="./input/mi350_4nic_config.json"
+          fi
```

could we perhaps do something like this?

```suggestion
         CONFIG_FILE = "./input/mi350_${ENABLED_NICS}_config.json"
```

since there will always be a value?